README.md

```diff
@@ -2,21 +2,38 @@
 
 Create a tag and an optional release
 
-## Usage
+### Usage
```

Comment:
```suggestion
## Usage
```
This is the first heading after the 1st level heading which makes it a 2nd level heading. Please also adjust the other headings below as appropriate.